examples/server-side-rendering/src/createApp.js

```diff
@@ -1,13 +1,14 @@
 import algoliasearch from 'algoliasearch/lite';
+
 import App from './App';
 
+const searchClient = algoliasearch(
```

Comment:
Is this change still required?

Reply:
Yeah - customers usually put the algoliasearch outside of the scope of `createApp`. So it's instance can be shared between requests.

Also, the that's the way we document things.